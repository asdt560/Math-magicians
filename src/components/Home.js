import React from 'react';

function Home() {
  return (
    <section className="homecontainer">
      <h2>Welcome to our page</h2>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit.
        Fuga, perspiciatis. Iusto, saepe? Provident eveniet ducimus sit
        quod aperiam, molestias, modi quis voluptates delectus voluptate
        sint consequatur voluptatibus nulla dicta veniam?
        Lorem ipsum dolor sit amet, consectetur adipisicing elit.
        Dolorem, minus sit placeat earum omnis sint alias rerum enim dignissimos
        dolorum deleniti blanditiis voluptatem distinctio eveniet aliquam.
        Quod, aut minus. Alias?
      </p>
      <p>
        Lorem ipsum dolor sit amet consectetur adipisicing elit.
        Mollitia nihil maxime rem voluptas sed hic doloribus
        numquam asperiores provident temporibus vero omnis,
        illo neque placeat molestias ad dolores non! Beatae.
        Lorem ipsum dolor sit amet consectetur adipisicing elit.
        Fuga magnam sequi veniam, ad error quae dicta vitae quibusdam,
        eligendi deleniti optio quidem laborum, sint repudiandae accusantium.
        Sunt quidem dolorum ipsum.
      </p>
    </section>
  );
}

export default Home;
