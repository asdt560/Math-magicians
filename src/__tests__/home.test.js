import '@testing-library/jest-dom';
import React from 'react';
import { render, screen } from '@testing-library/react';
import Home from '../components/Home';

test('To test for display', () => {
  render(<Home />);
  const tree = screen.getByText('Welcome to our page');
  expect(tree).toMatchSnapshot();
});
