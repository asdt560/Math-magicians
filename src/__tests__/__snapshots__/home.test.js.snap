// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`To test for display 1`] = `
<h2>
  Welcome to our page
</h2>
`;
